demo/styling.html

```diff
@@ -25,6 +25,12 @@ <h3>Styling</h3>
       To stay on a safe side, only apply custom styles to either the element
       itself (<code>:host</code>) or the exposed shadow parts (<code>[part~="name"]</code>).
     </p>
+
+    <p>
+      <a href="../#/elements/vaadin-text-field#description" target="_parent">
+        <strong>List of all styling parts ↗</strong>
+      </a>
+    </p>
     <p>
       <b>NOTE: The custom style module must be defined/imported before the element
       it targets.</b>
```

Comment:
Btw, this is no longer necessary. The order of imports doesn’t matter, as long as the style module is imported before the element is upgraded.